docker-compose.yml

```diff
@@ -1,32 +1,42 @@
 version: "3.9"
 
 services:
-  app:
-    image: multilateralfund:latest
-    build:
-      context: .
-      dockerfile: Dockerfile
-    ports:
-      - 8000:8000
+  postgres:
+    image: postgres:15
+    environment:
+      - POSTGRES_DB
+      - POSTGRES_USER
+      - POSTGRES_PASSWORD
     volumes:
-      - .:/var/local/multilateralfund
+      - pg-data:/var/lib/postgresql/data
+    expose:
+      - 5432
+    restart: always
+
+  app:
+    image: ghcr.io/multilateralfund/multilateralfund-app:${DOCKER_TAG:-main}
     env_file:
       - .env
     environment:
       - DJANGO_MIGRATE=yes
-    command: [ "run" ]
+    volumes:
+      - app_data:/app
+    expose:
+      - 8000
     depends_on:
       - postgres
 
-  postgres:
-    image: postgres:15
-    environment:
-      - POSTGRES_DB
-      - POSTGRES_USER
-      - POSTGRES_PASSWORD
+  nginx:
+    image: ghcr.io/multilateralfund/multilateralfund-nginx:${DOCKER_TAG:-main}
     volumes:
-      - pg-data:/var/lib/postgresql/data
+      - app_data:/app
```

Comment:
Same here, an app volume doesn't seem necessary. I would suggest replacing it with a volume for the static files (and maybe media/uploaded files if that's something that's going to be added in the future in the project)